u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java

```diff
@@ -40,11 +42,7 @@ public TokenStorageData(SecurityKeyData tokenData) {
     this.enrollmentTime = tokenData.getEnrollmentTime();
     this.keyHandle = tokenData.getKeyHandle();
     this.publicKey = tokenData.getPublicKey();
-    try {
-      this.attestationCert = tokenData.getAttestationCertificate().getEncoded();
-    } catch (CertificateEncodingException e) {
-      throw new RuntimeException();
-    }
+    this.attestationCert = X509Util.encodeCertArray(tokenData.getAttestationCertificateChain());
```

Comment:
Nit: this is no longer just a cert. Could you use a longer name, e.g. attestationCertChain? attestationCertOrCertAndChain? Ick.